### PR TITLE
Converted samples and test projects to netcoreapp2.0

### PR DESCRIPTION
**Pull request by kichalla**

### CODE REVIEW

File: Logging.sln [hunk 1]
```diff
@@ -1,42 +1,50 @@
...
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Microsoft.Extensions.Logging.EventSource", "src\Microsoft.Extensions.Logging.EventSource\Microsoft.Extensions.Logging.EventSource.csproj", "{84073E58-1802-4525-A9E5-1E6A70DAF0B2}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Extensions.Logging.EventSource", "src\Microsoft.Extensions.Logging.EventSource\Microsoft.Extensions.Logging.EventSource.csproj", "{84073E58-1802-4525-A9E5-1E6A70DAF0B2}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Microsoft.Extensions.Logging.EventSource.Test", "test\Microsoft.Extensions.Logging.EventSource.Test\Microsoft.Extensions.Logging.EventSource.Test.csproj", "{F3B898C3-D441-4207-A92B-420D6E73CA5D}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Extensions.Logging.EventSource.Test", "test\Microsoft.Extensions.Logging.EventSource.Test\Microsoft.Extensions.Logging.EventSource.Test.csproj", "{F3B898C3-D441-4207-A92B-420D6E73CA5D}"
```
Comment (natemcmaster) [inline Logging.sln]: Interesting. Did VS change this GUID? This changes how VS loads the project.

Comment (kichalla) [inline Logging.sln]: yeah VS changed this GUID. probably because the version of VS changed from `15.0.26127.0` to `15.0.26228.9`?

Comment (natemcmaster) [inline Logging.sln]: Interesting. Well, looks right: cref: https://github.com/dotnet/roslyn-project-system/blob/f71940e85c15f127a1e981db158dbb8296c57003/src/Microsoft.VisualStudio.ProjectSystem.CSharp.VS/Packaging/CSharpProjectSystemPackage.cs#L37-L38

Comment (natemcmaster) [inline Logging.sln]: https://github.com/dotnet/roslyn-project-system/issues/1821

**Comment by kichalla** [comment]
🆙 📅

**Comment by kichalla** [comment]
7172212fad9b794ff16323412f72d41601710b4f

**Pull request closed without merging**